{"uuid": "57329212-559d-4faf-be61-b6033b8aa593", "children": ["58a19f65-1cd6-45b8-951f-1cdaee9ab4ef"], "befores": [{"name": "one_time_setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x0x116c333+65459]\n\tGetHandleVerifier [0x0x116c374+65524]\n\t(No symbol) [0x0xf8d973]\n\t(No symbol) [0x0xfd76e7]\n\t(No symbol) [0x0xfd7a8b]\n\t(No symbol) [0x0x101dea2]\n\t(No symbol) [0x0xff9e44]\n\t(No symbol) [0x0x101b606]\n\t(No symbol) [0x0xff9bf6]\n\t(No symbol) [0x0xfcb38e]\n\t(No symbol) [0x0xfcc274]\n\tGetHandleVerifier [0x0x13eeda3+2697763]\n\tGetHandleVerifier [0x0x13e9ec7+2677575]\n\tGetHandleVerifier [0x0x1194194+228884]\n\tGetHandleVerifier [0x0x11849f8+165496]\n\tGetHandleVerifier [0x0x118b18d+192013]\n\tGetHandleVerifier [0x0x11747d8+99416]\n\tGetHandleVerifier [0x0x1174972+99826]\n\tGetHandleVerifier [0x0x115ebea+10346]\n\tBaseThreadInitThunk [0x0x76715d49+25]\n\tRtlInitializeExceptionChain [0x0x77ded6db+107]\n\tRtlGetAppContainerNamedObjectPath [0x0x77ded661+561]\n\n", "trace": "  File \"C:\\Development\\Projects\\Portfolio\\PytestWebAutomation\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Development\\Projects\\Portfolio\\PytestWebAutomation\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Development\\Projects\\Portfolio\\PytestWebAutomation\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Development\\Projects\\Portfolio\\PytestWebAutomation\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Development\\Projects\\Portfolio\\PytestWebAutomation\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Development\\Projects\\Portfolio\\PytestWebAutomation\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 930, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Development\\Projects\\Portfolio\\PytestWebAutomation\\tests\\punch_in_out_test.py\", line 11, in one_time_setup\n    login_page.login_user()\n    ~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Development\\Projects\\Portfolio\\PytestWebAutomation\\pages\\login_page.py\", line 12, in login_user\n    self.username_input.input(\"Admin\")\n    ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^\n  File \"C:\\Development\\Projects\\Portfolio\\PytestWebAutomation\\framework\\fields\\textbox_field.py\", line 22, in input\n    self.get_element().send_keys(text)\n    ~~~~~~~~~~~~~~~~^^\n  File \"C:\\Development\\Projects\\Portfolio\\PytestWebAutomation\\framework\\fields\\base_field.py\", line 25, in get_element\n    current_element = self.wait(current_element).until(EC.presence_of_element_located(locator))\n  File \"C:\\Development\\Projects\\Portfolio\\PytestWebAutomation\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 138, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1759012990685, "stop": 1759013001440}], "start": 1759012990685, "stop": 1759013001616}